src/engine/source/builder/builders/opBuilderHelperMap.cpp

```diff
@@ -947,7 +947,7 @@ base::Expression opBuilderHelperRegexExtract(const std::any& definition)
     if (!regex_ptr->ok())
     {
         throw std::runtime_error(
-            fmt::format("\"{}\" function: Error compiling regex \"{}\": {}",
+            fmt::format(R"("{}" function: Error compiling regex "{}": {})",
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C-plus-plus%29#modern-language-features-and-best-practices

Raw strings